File: 800/2130A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int t;
    cin >> t;

    while (t--) 
    {
        int n;
        cin >> n;

        int zero = 0;
        long long sum = 0;  // use long long in case values are large

        for (int i = 0; i < n; i++) 
        {
            int a;
            cin >> a;

            if (a == 0)
                zero++;
            
            sum += a;
        }

        cout << sum + zero << "\n";
    }

    return 0;
}

File: 800/2130A/main.java
import java.util.*;
public class main
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        
        while(t-- > 0)
        {
            int n = sc.nextInt();
            int zero = 0;
            int sum = 0;
            
            for(int i=0;i<n;i++)
            {
                int a = sc.nextInt();
                if(a == 0)
                    zero++;
                
                sum += a;
            }
            
            System.out.println(sum+zero);
        }
    }
}